src/components/Thumbnail/Name.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from "react";
+
+interface IProps {
+  children: string;
```

Comment:
this isn't a great practice, as children could be any renderable node
write:
```
import React, { FC, ReactNode } from "react";

interface IProps {
  children: ReactNode;
}
```
read for understanding https://medium.com/@martin_hotell/react-children-composition-patterns-with-typescript-56dfc8923c64